Runtime/Tracking/Collision/CollisionNotifier.cs

```diff
@@ -80,7 +80,7 @@ public bool Equals(EventData other)
                     return true;
                 }
 
-                return Equals(ColliderData.GetContainingTransform(), other.ColliderData.GetContainingTransform()) && ((ForwardSource == null && other.ForwardSource == null) || Equals(ForwardSource, other.ForwardSource));
+                return Equals(ColliderData, other.ColliderData) && ((ForwardSource == null && other.ForwardSource == null) || Equals(ForwardSource, other.ForwardSource));

```

Comment:
```suggestion
                return Equals(ForwardSource, other.ForwardSource) && Equals(ColliderData, other.ColliderData);
```
The `ReferenceEquals` above and the way `Equals` work already handle `null`-cases.